<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APP</class>
 <widget class="QMainWindow" name="APP">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>APP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="fdm_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>FDM</string>
    </property>
    <widget class="QPushButton" name="kaizen_pb">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>KAIZEN</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="generous_pb">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>GENEROUS</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="bride_pb">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>BRIDE</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dlp_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>431</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>DLP</string>
    </property>
    <widget class="QPushButton" name="queen_pb">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>QUEEN</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>771</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>771</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>281</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>firmware</string>
    </property>
    <widget class="QPushButton" name="raspberrypi_pb">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>Raspberry Pi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="arduino_pb">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>Arduino</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lcd_pb">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>LCD</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
